onnxruntime/core/providers/cpu/math/cumsum.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include "core/providers/cpu/tensor/pad.h"
+
+namespace onnxruntime {
+
+template <class T>
+class CumSum final : public OpKernel {
+ public:
+  explicit CumSum(const OpKernelInfo& op_kernel_info);
+
+  Status Compute(OpKernelContext* p_op_kernel_context) const override;
+
+ private:
+  int64_t _exclusive;
```

Comment:
exclusive_ and reverse_ (naming convention - trailing underscore)

Reply:
> exclusive_ and reverse_ (naming convention - trailing underscore)

Renamed with trailing underscore.